backend/test/integrated/group.js

```diff
@@ -114,7 +115,8 @@ describe("Groups", function () {
 		it("using master head revision should succeed", function(done){
 			agent.get(`/${username}/${model}/revision/master/head/groups/${groupID}`)
 				.expect(200 , function(err, res) {
-					expect(res.body).to.deep.equal(goldenData);
+					expect(res.body.color).to.deep.equal(goldenData.color);
```

Comment:
why do we have to split this up? result should match the goldenData entirely, if it doesn't then something sounds wrong.

Reply:
It returns extra things like _v